cg/store/api/find_business_data.py

```diff
@@ -746,3 +746,33 @@ def get_not_analysed_cases_by_sample_internal_id(
             filter_functions=[SampleFilter.FILTER_BY_INTERNAL_ID],
             internal_id=sample_internal_id,
         ).all()
+
+    def get_sample_by_internal_id(self, internal_id: str) -> Optional[Sample]:
+        """Return a sample by lims id."""
+        return apply_sample_filter(
+            filter_functions=[SampleFilter.FILTER_BY_INTERNAL_ID],
+            samples=self._get_query(table=Sample),
+            internal_id=internal_id,
+        ).first()
+
+    def metric_exists_in_status_db(self, metric: SampleLaneSequencingMetrics) -> bool:
```

Comment:
To keep the conventions of fucntion names starting with a verb
```suggestion
    def is_metric_in_status_db(self, metric: SampleLaneSequencingMetrics) -> bool:
```

Reply:
We should fix this 👍 I'd prefer to do it in a separate PR though since both moving stuff and renaming might end up a bit messy